.travis.yml

```diff
@@ -6,5 +6,4 @@ node_js:
   - "8"
 script:
   - npm install -g typescript
-  - cd run-suite-task && npm install . && cd ..
-  - bin/test.sh
+  - cd ./run-suite-task && npm install . && tsc && npm test
```

Comment:
Would it be clearer to have these commands as individual lines versus `&&`'ing them (or is that required because of the directory change)?

Reply:
I'm actually not sure what Travis is going to do here, without "testing" it I felt this was safe enough...

Reply:
Yea, no biggie -- Go with what you got